File: java/src/main/java/br/com/acqio/dto/TransactionResponse.java
package br.com.acqio.dto;

import br.com.acqio.enums.CardApplication;
import br.com.acqio.enums.PaymentStatus;
import lombok.*;

@Getter
@Setter
@ToString
@NoArgsConstructor(access = AccessLevel.PUBLIC)
public class TransactionResponse {
    private Long transactionId;
    private Integer status;
    private PaymentStatus paymentStatus;
    private String nsu;
    private String nsuOriginal;
    private String cardHiddenPan;
    private String authorizationCode;
    private String cardProductName;
    private String merchant;
    private String client;
    private String merchantHtml;
    private String clientHtml;
    private CardApplication cardApplication;
    private String cardHolderName;
}

File: csharp/csharp/dto/TransactionResponse.cs
using csharp.enums;
using System;

namespace csharp.dto
{
    class TransactionResponse
    {
        public long transactionId { get; set; }
        public int status { get; set; }
        public PaymentStatus paymentStatus { get; set; }
        public String nsu { get; set; }
        public String nsuOriginal { get; set; }
        public String cardHiddenPan { get; set; }
        public String authorizationCode { get; set; }
        public String cardProductName { get; set; }
        public String merchant { get; set; }
        public String client { get; set; }
        public String merchantHtml { get; set; }
        public String clientHtml { get; set; }
        public CardApplication cardApplication { get; set; }
        public String cardHolderName { get; set; }
    }
}